src/cleanvision/issue_managers/image_property_issue_manager.py

```diff
@@ -15,6 +15,43 @@
 from cleanvision.utils.base_issue_manager import IssueManager
 from cleanvision.utils.constants import IMAGE_PROPERTY
 
+import multiprocessing
+from cleanvision.issue_managers.image_property import (
+    calc_brightness,
+    calc_aspect_ratio,
+    calc_entropy,
+    calc_blurriness,
+    calc_color_space,
+)
+
+# psutil is a package used to count physical cores for multiprocessing
+# This package is not necessary, because we can always fall back to logical cores as the default
+try:
+    import psutil
+
+    psutil_exists = True
+except ImportError as e:  # pragma: no cover
+    psutil_exists = False
+
+
+def compute_scores(arg: Dict[str, Any]) -> Dict[str, Any]:
+    compute_functions = {
```

Comment:
Pass `compute_functions` dict as an argument

Reply:
does this make a big difference? I'm in favor of keeping this as is, instead of passing it successively through the compute function and the wrapper, because it is not used anywhere else

Reply:
I agree passing the dict is an overhead, can you make it a global variable in the module and access it? Since it is a static object, it doesn't make sense to have it in the method.

Reply:
There's actually something subtle here: your approach of making it a global variable would work *only on Linux*, because only there are sub-processes started with the fork method.

On MacOS and Windows, processes are started with spawn, so they would not have access to global variables. In that case, we would need to pass in these functions as inputs to the subprocesses, so the current code would be necessary anyway. Thus I still think the best is just to leave as is.

Reply:
I see, thanks for explaining.